js_modules/dagster-ui/packages/ui-core/src/selection/BaseSelectionVisitor.ts

```diff
@@ -0,0 +1,154 @@
+import {ParserRuleContext} from 'antlr4ts';
```

Comment:
This file was extracted from the existing SelectionVisitor. none of the logic was changed